x-pack/test/screenshot_creation/apps/ml_docs/anomaly_detection/generate_anomaly_alerts.ts

```diff
@@ -105,13 +106,14 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await ml.navigation.navigateToAlertsAndAction();
         await pageObjects.triggersActionsUI.clickCreateAlertButton();
         await ml.alerting.setRuleName('test-ecommerce');
-
-        await ml.alerting.openNotifySelection();
+        const searchBox = await testSubjects.find('ruleSearchField');
+        await searchBox.click();
+        await searchBox.clearValue();
+        await searchBox.type('ml');
+        await searchBox.pressKeys(ENTER_KEY);
```

Comment:
I think using the browser service instead of the hard-coded key value is more stable, so I'd suggest to keep the `const browser = getService('browser');` and adjust here:
```suggestion
        await searchBox.pressKeys(browser.keys.ENTER);
```